Patches.py

```diff
@@ -2221,6 +2221,10 @@ def update_scrub_text(message: bytearray, text_replacement: list[str], default_p
     # Meg respawns after 30 frames instead of 100 frames after getting hit
     rom.write_byte(0xCDA723, 0x1E)
 
+    # Boss doors side range (1.0 value is 0x46)
+    # This was reduced to 0x32 in 1.1, likely to fix the Phantom Ganon door bug.
```

Comment:
This could use some elaboration, or maybe just a link to this PR?

Reply:
How about 
```
# Boss doors side range (1.0 value is 0x46)
# This was reduced to 0x32 in 1.1, either to fix the Phantom Ganon door bug or just to match better visually the door textures.
# See https://github.com/OoTRandomizer/OoT-Randomizer/pull/2331 for more information.
```

Reply:
Sounds good.